src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.fileservices.CreateReportFile;
+import core.basesyntax.service.fileservices.ParseFile;
+import core.basesyntax.service.fileservices.ReadFileImpl;
+import core.basesyntax.service.summaryofoperations.GeneralCalculation;
+import core.basesyntax.service.summaryofoperations.PreparationReportList;
+import core.basesyntax.storage.FruitStorage;
+import core.basesyntax.storage.PrepareList;
+import core.basesyntax.storage.ProcessingData;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String inputFilePath = "src/main/java/core/basesyntax/resources"
+            + "/FruitFiles.csv";
+    private static final String reportFilePath = "src/main/java/core/basesyntax/resources"
+            + "/reportFiles/report.csv";
+    private static final Map<Operation, Map<String, List<Integer>>> fruitMaps = new HashMap<>();
+
+    public static void main(String[] args) {
+        ParseFile parseFile = new ParseFile();
+        String filePath = parseFile.parseFileWithData(inputFilePath);
+        ReadFileImpl readFile = new ReadFileImpl(parseFile);
+        List<String> listOfFruits = readFile.readFromCsvFile(filePath);
```

Comment:
verbs are used in method names, usage of them in variables/classes names creates lots of confusion during code reading
```suggestion
        FileParser fileParser = new FileParser();
        String filePath = fileParser.parseFileWithData(inputFilePath);
        FileReaderImpl fileReader = new FileReaderImpl(fileParser);
        List<String> listOfFruits = fileReader.readFromCsvFile(filePath);
```
Please recheck all naming